internal/discoverer/nacos_test.go

```diff
@@ -0,0 +1,193 @@
+package discoverer
+
+import (
+	"fmt"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/api7/apisix-seed/internal/conf"
+	"github.com/api7/apisix-seed/internal/core/comm"
+	"github.com/api7/apisix-seed/internal/utils"
+	"github.com/nacos-group/nacos-sdk-go/clients"
+	"github.com/nacos-group/nacos-sdk-go/clients/naming_client"
+	"github.com/nacos-group/nacos-sdk-go/common/constant"
+	"github.com/nacos-group/nacos-sdk-go/vo"
+	"github.com/stretchr/testify/assert"
+)
+
+var nacosConf interface{}
+var registerClient naming_client.INamingClient
+var TestService string
+var TestGroup string
+
+func init() {
+	bc := ReadFile("discoverer.yaml")
```

Comment:
Could you provide unit tests for hosts with https/port/auth?

Reply:
For https and port testing is possible, but for auth there is no official sample server to test.

Reply:
Actually, I mean unit test. We can verify if some behavior is expected, like the argument we provide to Nacos Client API.

Reply:
I have currently tested the server config and can verify the process of https/auth/port arguments extraction.  

For the client config, due to the current implementation of Nacos, it will not be available after the client is generated. I have submitted a PR to try to fix this issue. If it works, tests for client config can be added later.